scripts/dev-restart.sh

```
@@ -0,0 +1,172 @@
+fi
+
+# 5. Clear cache dan build artifacts
+echo "\n🗑️  Clearing cache and build artifacts..."
+rm -rf apps/frontend/.next 2>/dev/null || true
+rm -rf apps/backend/dist 2>/dev/null || true
+rm -rf apps/frontend/.turbo 2>/dev/null || true
+rm -rf apps/backend/.turbo 2>/dev/null || true
+
+# 6. Start backend server
+echo "\n🚀 Starting backend server on port $BACKEND_PORT..."
+cd apps/backend
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)